design-library/source/_patterns/01-atoms/08-logo/logo.md

```diff
@@ -0,0 +1,6 @@
+# Atom: List Title
+
+*Belongs to*
+
+* Molecule: List Element
```

Comment:
I don't think this "belongs to" is correct

Reply:
Quite right!